WcaOnRails/db/migrate/20200505175327_create_delegates_table.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class CreateDelegatesTable < ActiveRecord::Migration[5.2]
+  def change
+    create_table :delegates do |t|
+      t.references :user, null: false
+      t.string :status, null: false
+      t.references :region, null: false
+      t.references :subregion
+      t.references :country
```

Comment:
Can we have the actual column created by this? (that's why I wanted to see a draft with the actual column name ;))
I'm afraid this will create a `countryId` column, which, despite its name, is not a reliable id.
We absolutely need to use `country_iso2` here.